dev-libs/skalibs/skalibs-2.8.1.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="skarnet.org general-purpose libraries"
+HOMEPAGE="https://www.skarnet.org/software/skalibs/"
+SRC_URI="https://www.skarnet.org/software/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="doc ipv6 static-libs"
+
+RDEPEND=""
+DEPEND=""
+
+DOCS="AUTHORS NEWS README*"
+HTML_DOCS="doc/*"
+
+src_prepare() {
+	default
+
+	# Remove QA warning about LDFLAGS addition
+	sed -i "s/tryldflag LDFLAGS_AUTO -Wl,--hash-style=both/:/" "${S}/configure" || die
```

Comment:
`/foo/d` is the usual way, presuming it's on separate line.

Reply:
In this case, it's the only line inside an `if` statement, so it's invalid syntax to simply remove the line.